GLD/PHL/PHL_1997_LFS/PHL_1997_LFS_v01_M_v01_A_GLD/Programs/PHL_1997_LFS_v01_M_v01_A_GLD_ALL.do

```diff
@@ -1017,7 +1017,7 @@ foreach v of local ed_var {
 				* the string variable in isic4 will is industrycat_isic
 
 	// replace one code that I know doesn't match
-	rename 		isic3_1_2dig	isic3_1_2dig_`n'
+	rename 		isic3_1_2dig_pad	isic3_1_2dig_`n'
```

Comment:
here's an example of the varname substitution after the key merge that is conceptually applied to all GLD code in relevant key merges. Note that this is not relevant in situations where the key contains 4-digit information.